src/src/TemplateBundle/Controller/AssetController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace TemplateBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+
+class AssetController extends CustomTemplateController
+{
+    /**
+     * @Route("/asset/{templateName}/css/{assetName}", name="get_css", defaults={"assetName": "styles"})
+     * @Method("GET")
+     */
+    public function getCssAction($templateName, $assetName, Request $request)
+    {
+        if (empty($assetName)) {
+            $assetName = 'styles';
+        } elseif (false !== strpos($assetName, '..') || false !== strpos($assetName, '/')) {
+            throw new \Exception('Forbidden characters used as a asset name.');
```

Comment:
Nice, na toto by som ja asi nemyslel.

Reply:
Co som to testoval tak to uz zachyti aj ta twig extension, ale priemym volanim na controller by sa to stale dalo obist.